src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/IHttpHeadersHandler.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Internal.Http
+{
+    public interface IHttpHeadersHandler
```

Comment:
`IHttpHeadersHandler` and `IHttpRequestLineHandler` are used as a generic constraint for `IHttpParser` should just be direct call in use?